src/AnyChart/components/AnyChart.ts

```diff
@@ -29,6 +30,12 @@ export type AnyChartProps = AnyChartComponentProps & typeof PlotlyChartActions &
 class AnyChart extends Component<AnyChartProps> {
     private tooltipNode?: HTMLDivElement;
 
+    shouldComponentUpdate(nextProps: Readonly<AnyChartProps>): boolean {
+        // TODO: this version of TypeScript and the types from @types/deep-equal are not compatible. As an interim solution, the compilation error is ignored. Either fix typings/deep-equal or upgrade TS.
+        // @ts-ignore
```

Comment:
Nit: I prefer `@ts-expect-error` since that doesn't alter compilation (got that warning a few times)

Reply:
It looks like `@ts-expect-error` was introduced in [3.9](https://www.typescriptlang.org/docs/handbook/release-notes/typescript-3-9.html#ts-ignore-or-ts-expect-error). This repo is 2.8.